src/components/Tour/TourBox.tsx

```diff
@@ -0,0 +1,29 @@
+import { Flex, useThemeContext } from "@radix-ui/themes";
+import { CSSProperties, ReactNode } from "react";
+
+export type TourBubbleProps = {
+  children?: ReactNode;
+  style?: CSSProperties;
+};
+
+export function TourBox({ children, style }: TourBubbleProps) {
+  const appearance = useThemeContext().appearance;
```

Comment:
There's a hook for appearance, that also watches for changes from `vscode` in the hooks directory